src/Umbraco.Web.UI.Client/src/common/services/validationhelper.service.js

```diff
@@ -0,0 +1,34 @@
+(function () {
+    'use strict';
+
+    function validationHelperService($q, localizationService) {
```

Comment:
We are trying to avoid the term `helper` in our code. We have quite a lot of code dealing with validation and `validationHelper` is too generic. Is there a different/more specific name for this service we could use? Or can this method exist on an already existing service?

Reply:
Updated to a more specific `validationMessageService`.

Reply:
Small thing - but the file name will need to be change too,

Reply:
Updated.